File: app/src/androidTest/java/ru/roscha_akademii/medialib/common/MockAppRunner.java
package ru.roscha_akademii.medialib.common; // MockAppRunner

import android.app.Application;
import android.content.Context;
import android.support.test.runner.AndroidJUnitRunner;

public class MockAppRunner extends AndroidJUnitRunner {
    @Override
    public Application newApplication(ClassLoader cl, String className, Context context)
            throws InstantiationException, IllegalAccessException, ClassNotFoundException
    {
        return super.newApplication(cl, MockMediaLibApplication.class.getName(), context);
    }
}

File: app/src/androidTest/java/ru/roscha_akademii/medialib/common/MockAppRunner.kt
package ru.roscha_akademii.medialib.common

// MockAppRunner

import android.app.Application
import android.content.Context
import android.support.test.runner.AndroidJUnitRunner

class MockAppRunner : AndroidJUnitRunner() {
    override fun newApplication(cl: ClassLoader, className: String, context: Context): Application {
        return super.newApplication(cl, MockMediaLibApplication::class.java.name, context)
    }
}

